README.md

```diff
@@ -0,0 +1,19 @@
+**MEI 3 to 5**
+=====================================
+
+This repository contains a collection of scripts designed to update MEI 3 files to MEI 5. The current implementation of this repository only supports upgrading MEI files from the Liber Usualis. Future developments may extend this functionality to support additional MEI files.
+
+
+**Prerequisites**
+--------------------------------
+
+The only required dependency is the lxml library. You can i
+
+**Usage**
+-----
+
+1. If you do not have it already, install lxml using `pip install lxml`
+2. Download the scripts in this repository into the same directory as your MEI files
```

Comment:
I think better would be to have people clone this repository locally and then put the MEI files into that directory (or move the scripts into the same directory as files post-cloning).

Reply:
Fixed